chaincfg/params.go

```diff
@@ -155,14 +165,29 @@ var MainNetParams = Params{
 	Name:        "mainnet",
 	Net:         wire.MainNet,
 	DefaultPort: "8333",
-	DNSSeeds: []string{
-		"seed.bitcoin.sipa.be",
-		"dnsseed.bluematt.me",
-		"dnsseed.bitcoin.dashjr.org",
-		"seed.bitcoinstats.com",
-		"seed.bitnodes.io",
-		"bitseed.xf2.org",
-		"seed.bitcoin.jonasschnelli.ch",
+	DNSSeeds: []DNSSeed{
+		{
+			"seed.bitcoin.sipa.be",
```

Comment:
I think these will comfortably fit on a single line each and would look cleaner.

```Go
DNSSeeds: []DNSSeed{
	{"seed.bitcoin.sipa.be", true},
	{"dnsseed.bluematt.me", true},
	{"dnsseed.bitcoin.dashjr.org", false},
	{"seed.bitcoinstats.com", true},
	...
}